script/Ciel/Day 12/12_01_CIEL12_1.txt

```diff
@@ -1,7 +1,8 @@
 [sha:902b4d50071a15718e21c3d035a0a6f7624a459f]{
 -- Page 1, Offset 30866.
 -- 「――――――！」
-"―――!"
+// lint-off:LintEmDashes
+"――――――!"
```

Comment:
Shocked gasp; maybe 
>Whoa!

or maybe

>*Gasp*

idk

Reply:
going with the latter

Reply:
on second thought, I opted for something different